### PR TITLE
mvcc: simplify watchable_store addVictim code

### PR DESCRIPTION
**Pull request by lx1036**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: server/storage/mvcc/watchable_store.go [hunk 1]
```diff
@@ -458,7 +452,7 @@ func (s *watchableStore) notify(rev int64, evs []mvccpb.Event) {
 }
 
 func (s *watchableStore) addVictim(victim watcherBatch) {
-	if victim == nil {
+	if victim == nil || len(victim) == 0 {
```
Comment (serathius) [inline server/storage/mvcc/watchable_store.go:455]:
```suggestion
	if len(victim) == 0 {
```
Don't think that checking nil is needed here.

Comment (lx1036) [reply server/storage/mvcc/watchable_store.go]: resolved, thanks

**Review by serathius — APPROVED** [review]
Looks good, thanks for contributing! Making code cleaner and more understandable is as much or more important than adding features.

**Review by spzala — APPROVED** [review]
lgtm
Thanks @lx1036

**Pull request merged**